source/config.hpp

```diff
@@ -52,6 +52,7 @@ class Config
 	string default_member_rvalue_reference_return_value_policy_ = "pybind11::return_value_policy::automatic";
 	string default_call_guard_ = "";
 	string holder_type_ = "std::shared_ptr";
+	string include_file_ = "pybind11/pybind11.h";
```

Comment:
Could you please rename this to something like `pybind11_include_file_`? Otherwise name will be quite confusing (ie include file for what?) since we talk about includes all over Binder code. Thanks,